src/file.toit

```diff
@@ -567,5 +568,47 @@ update-time path/string --access/Time?=null --modification/Time?=null:
 
   update-times_ path atime-s atime-ns mtime-s mtime-ns
 
+/**
+Searches for an executable file with the given $name in the system PATH.
+Returns null if no such executable is found.
+Returns the path as provided by the PATH. For example, if the PATH contains
```

Comment:
"Returns the path as provided by the PATH." is difficult to understand. The example is fine. Maybe something like:

Returns a path relative to the part of the PATH where the executable file is found.